pkg/loop/internal/core/services/capability/capabilities_registry_test.go

```diff
@@ -153,6 +169,65 @@ func TestCapabilitiesRegistry(t *testing.T) {
 	_, err = rc.Get(tests.Context(t), "some-id")
 	require.ErrorContains(t, err, "capability not found")
 
+	pid := p2ptypes.PeerID(randomWord())
+	node := capabilities.Node{
+		PeerID: &pid,
+		WorkflowDON: capabilities.DON{
+			ID: "workflow-don-id",
+			Members: []p2ptypes.PeerID{
+				randomWord(),
```

Comment:
/nit if there is no explicit utility in the randomness of these values then i would prefer determenisitic tests.

note: you can easily allocate bytes by assigning values at indices
[]byte{0:1, 31:2} //sets byte 0 to `1` and byte 31 to `2`
https://go.dev/play/p/pCSesxuK8GS

Reply:
there isnt, i just copied from tests in the chainlink repo that do the same thing